themes/bootstrap3/templates/ajax/resultgooglemapinfo.phtml

```diff
@@ -7,6 +7,7 @@
       <div class="mapInfoResult <?php if ($i % 2 == 0): ?>alt <?php endif; ?>record<?=$i ?>">
         <div class="mapInfoResultThumb">
           <?php if ($thumb = $this->record($record)->getThumbnail()): ?><img class="mapInfoResultThumbImg" src="<?=$this->escapeHtmlAttr($thumb) ?>"/><?php endif; ?>
+          <?=$this->ajaxCovers()->detail('small')?>
```

Comment:
I see that you've added ajaxCovers support to a lot of templates where thumbnails are rendered, and I imagine that some of these may be difficult to test without a fairly specific setup. It might be worthwhile to create a set of TODO checkboxes for testing some of the narrower edge cases. If there's anything you can't easily check yourself, please make a TODO box and I'll be happy to help with the testing.

Reply:
I added todo list for templates and will try to test them

Reply:
Thanks! Let me know if you need help and I'll take a closer look when I have a bit more time available.

Reply:
I'll try to test what I will be able and I want also create some unit test for ajax handler, after that, i will need you next review round

Reply:
Thanks, that sounds great; just let me know once you are ready.